### ISSUE TITLE
PDF version

### ISSUE THREAD

**Issue by eMPee584** [coodict/python3-in-one-pic#20]
Uhm.. png is nice but not as searchable, so how about a PDF version :cowboy_hat_face:

**Comment by lamjiidii1** [comment]
Very nice

**Comment by xcruiser** [comment]
yeah i was looking for PDF searchable, hope the creator will do another favor to the community 

THANK YOU
